tests/test_api.py

```diff
@@ -357,3 +359,23 @@ def test_export_cross_prediction():
     )
 
     assert isinstance(sorted_dict, dict)
+
+
+@pytest.mark.parametrize("test_df", test_dfs)
+def test_allocation_limit(test_df):
+    """Test used to see if calculated allocation values are inside of specified limit"""
+
+    test_df_copy = copy.deepcopy(test_df)
+    df_with_allocations = Api.calculate_allocations(
+        df=test_df_copy, name_of_strategy=available_allocation_algorithms[0], name_of_price_series="close",
+        allocation_lower_limit=0, allocation_upper_limit=0
+    )
+    if all(df_with_allocations["allocation"] != 0):
```

Comment:
```suggestion
    if not all(df_with_allocations["allocation"] == 0.0):
```

Reply:
Otherwise will not trigger unless all break the limits.

Reply:
I commited the recommended change from all to any, we could leave it like that or should I change it to "not all"